.gitignore

```diff
@@ -168,3 +168,4 @@ data
 # DS_Store files
 .DS_Store
 src/apis/uploads/
+src/DS_Store
```

Comment:
The DS-Store files should be excluded, but I can still see one DS_Store file inside the services directory and only, when i view the Pull-Request inside VS-Code. Is this just a Bug or does it still track these files?

Reply:
These files were already tracked by Git, so this line had no effect. I now removed them and adjusted the gitignore accordingly. Should be fine now 👍🏻